jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/main/java/org/jbpm/console/ng/pr/client/editors/instance/details/multi/ProcessInstanceDetailsMultiPresenter.java

```diff
@@ -248,17 +248,17 @@ public void callback(List<NodeInstanceSummary> completedNodes) {
                                                             .addParameter("completedNodes", completedNodeParam.toString())
                                                             .addParameter("readOnly", "true")
                                                             .addParameter("processId", processId)
-                                                            .addParameter("deploymentId", selectedDeploymentId)));
+                                                            .addParameter("processInstanceId", selectedDeploymentId)));
```

Comment:
Can't load the process instance diagram. Please review, I believe `deploymentId` is required.